pkg/common/config/types.go

```diff
@@ -89,4 +93,10 @@ type VirtualCenterConfig struct {
 	CAFile string `gcfg:"ca-file"`
 	// Thumbprint of the VCenter's certificate thumbprint
 	Thumbprint string `gcfg:"thumbprint"`
+	// IP Family enables the ability to support IPv4, IPv6, or a dual
+	// stack with priority given to a certain IP type ipv6,ipv4
```

Comment:
I think we should be clearer on the semantics of this config field. Since we only support "ipv4", "ipv6", "ipv4,ipv6" and "ipv6,ipv4" I think we can document that here directly.

Reply:
Sure. Will update.